runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -215,7 +215,7 @@ else if (config.clue() && message.toLowerCase().startsWith("!clues") && message.
 			log.debug("Running clue lookup for {}", search);
 			executor.submit(() -> playerClueLookup(setMessage, search));
 		}
-		else if (message.toLowerCase().startsWith("!kc "))
+		else if (config.killcount() && message.toLowerCase().startsWith("!kc"))
```

Comment:
Since the `startsWith()` check is changed, this should probably follow the example of the other commands and add a `message.length() > 4` check as well to guard against input like `!kcbandos`.